### PR TITLE
go/README: add a bit more info about mcp

### PR DESCRIPTION
**Pull request by andriytk**
-----
[View rendered bindings/go/README.md](https://github.com/Seagate/cortx-motr/blob/andriytk-patch-1/bindings/go/README.md)

### CODE REVIEW

**Comment by andriytk** [comment]
cc @chumakd

File: bindings/go/README.md [hunk 1]
```diff
@@ -46,9 +47,20 @@ Usage: mcp [options] src dst
...
 
 ```sh
-cd motr/bindings/go/mcp && go build
+cd motr/bindings/go/mcp && go build && go install
 ```
+
+If the build is successful, the binary will be installed
+to your `GOBIN` directory (check with `go env GOBIN`).
+
+See the usame example at this discussion thread -
```
Comment (huanghua78) [inline bindings/go/README.md:65]: typo.

s/usame/usage/

**Review by huanghua78 — CHANGES_REQUESTED** [review]
please fix the typo.

File: bindings/go/README.md (hunk 1 above)
Comment (andriytk) [reply bindings/go/README.md]: Fixed. Thanks!

**Comment by andriytk** [comment]
@huanghua78 good spot, Hua! As usual. Thanks! :)

**Comment by andriytk** [comment]
Good spot, @huanghua78. Thanks!

**Review by huanghua78 — APPROVED** [review]
Looks fine.

**Pull request merged**